JavaScript/JavaScriptSDK/Sender.ts

```diff
@@ -370,6 +370,18 @@ module Microsoft.ApplicationInsights {
             this._setupTimer();
         }
 
+        private _formatErrorMessage(xhr: XMLHttpRequest, xdr: XDomainRequest, message?: string): string {
+            if (xhr) {
```

Comment:
Minor: would it be preferable to have to methods, so that usage is a bit easier to read, e.g. formatErrorMessageXdr(), formatErrorMessageXhr()? Now it sumps around a bit like (null, xdr) vs. (xhr, null).